tests/python/pants_test/pantsd/pantsd_integration_test_base.py

```diff
@@ -47,7 +47,7 @@ def assert_started(self, timeout=16):
     self._check_pantsd_is_alive()
     return self._pid
 
-  def assert_pantsd_runner_started(self, client_pid, timeout=4):
+  def assert_pantsd_runner_started(self, client_pid, timeout=12):
```

Comment:
I also notice these kinds of timeouts being required locally, the pantsd integration tests would consistently fail whenever I ran them. So this increase seems to make sense.